src/bank_app/bank_app.py

```diff
@@ -0,0 +1,72 @@
+class BankAccount:
+    def __init__(self, balance: float, account_number: str, account_holder: str):
+        self.__balance = balance
+        self.__account_number = account_number
+        self.__account_holder = account_holder
+        self.__transaction_history = []
+        self.__overdraft_limit = 0
+
+    def get_balance(self):
+        return self.__balance
+
+    def add_balance(self, amount):
+        self.__balance += amount
+        self.__transaction_history.append(f"Deposited {amount}")
+        print(f"Deposited {amount}. Current balance: {self.__balance}")
+
+    def card_payment(self, price):
+
+        transaction = self.__balance - price
+
+        if transaction < 0:
+            raise ValueError("Transaction rejected. No funds in account.")
+
+        self.__balance -= price
+        self.__transaction_history.append(f"Withdraw {price}")
+        print(f"Withdraw {price}. Current balance: {self.__balance}")
+
+    def get_transaction_history(self):
+        return "\n".join(self.__transaction_history)
+
+    def transfer(self, amount, other_account):
+
+        if self.__balance < amount:
+            print("Transfer canceled. No funds in account.")
+            return
+
+        self.__balance -= amount
+        other_account.__balance += amount
+        self.__transaction_history.append(
+            f"Transferred {amount} to account {other_account.__account_number}"
+        )
+        other_account.__transaction_history.append(
+            f"Received {amount} from account {self.__account_number}"
+        )
+        print(
+            f"Transferred {amount} to account {other_account.__account_number}. Current balance: {self.__balance}"
+        )
+
+    def print_statement(self):
```

Comment:
Please add docstirngs to class method

Reply:
[Added docstrins in bank_app.py](https://github.com/Lukk1515/bank_app/pull/1/commits/2e50a2b4fe468c2774318e44a563e8d8d809876a)